app/pokedex/page.tsx

```diff
@@ -13,8 +13,7 @@ export default async function Pokedex() {
             ...pokemon,
             icon_url: getPokemonIconUrl(pokemon),
             full_image: getPokemonFullImageUrl(pokemon),
-            first_type_colour: `var(--type_${pokemon.type[0]})`,
-            second_type_colour: pokemon.type[1] ? `var(--type_${pokemon.type[1]})` : `var(--type_${pokemon.type[0]})`,
+            type_colours: [ `var(--type_${pokemon.type[0]})`, pokemon.type[1] ? `var(--type_${pokemon.type[1]})` : `var(--type_${pokemon.type[0]})` ],
```

Comment:
could move the pokemon data transformation logic into a helper function for clarity and reuseability

Reply:
Interesting idea. I applied this and also have it filter out type 2 if the pokemon is a monotype (would be null), and updated logic in the tile component to handle that case